src/sonic-config-engine/portconfig.py

```diff
@@ -151,21 +150,22 @@ def parse_port_config_file(port_config_file):
 # Generate configs (i.e. alias, lanes, speed, index) for port
 def gen_port_config(ports, parent_intf_id, index, alias_at_lanes, lanes, k,  offset):
```

Comment:
Let's change the "alias_at_lanes" to "alias_list" in the whole file.

Reply:
#done